### PR TITLE
Chrome: Overwrite existing window + bump to jquery 2 and chrome manifest v2

### PR DESCRIPTION
**Pull request by bhrt**
Hi,

First of all thanks for your work. The plugin is great!

I changed the chrome version a bit.
The main minor problem of Your plugin was that i couldn't keep my saved windows organised. Every saved window was a new entry so in time there was many different copies of the same window.

I added Overwrite saved window feature. 
It works like this.
If You have saved window with specific name, and you want to save new window with the same name (perhaps with different tabs opened). After you click on save button, plugin will ask you if you want to overwrite existing entry.
If you confirm, old entry is removed, and new one is saved with the old name.

It works good for me, so if You want to pull this feature feel free to do that.

I had to upgrade plugin manifest to version: 2 that's why i had to put all js code in external files.
I also bumped up versions of:
jquary to v2
jquaryui to newest.

PS. I know the code is hard to follow, due to huge changes. And i know that this could be done better, but that is what suits me, and i have no more time to spare ;)

Regards,
bhrt

### CODE REVIEW

**Comment by harshhpareek** [comment]
Hi, @bhrt . I think the original dev has left this project. I was looking at his license and you can take ownership of it and publish your version to the chrome web store. 

I could take ownership too, but I am not experienced with all the technologies in this project.

**Comment by bhrt** [comment]
I think he did.

Well, i'm not taking ownership of this repo due to exactly the same issues ;)
I made few changes cause lack of that functionality on chrome plugin was the issue for me, but i don't think i can develop whole project.
Feel free to use my mod. I hope You like it.

**Comment by Connorhd** [comment]
Hey, thanks for the contributions, I would love to work more on this project but I simply don't have time right now.

If anyone wants to fork and release alternative versions of the chrome extension that is totally fine with me.

**Comment by harshhpareek** [comment]
Do you think you could transfer ownership of the extension on the chrome
store?

**Comment by Connorhd** [comment]
No, you're welcome to publish your own version in the store though

**Comment by leni-kirilov** [comment]
@bhrt  small gradual changes would save you from having to write "sorry for the long PR" ;)

**Comment by bhrt** [comment]
Sorry for the long PR. :)
Yeah, I know how this looks like... but I haven't planned to share those changes.
